dts/bindings/input/futaba,sbus.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2024 NXP Semiconductors
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  SBUS input driver using
+  This driver implements the SBUS protocol used on RC radio's
+  to send out analogue joystick and switches output.
+  SBUS is an single-wire inverted serial protocol to either you need to
+  to the rx-invert feature of your serial driver or use an external signal inverter.
+  The driver binds this to the Zephyr input system using INPUT_EV_CODES.
+
+  The following examples defines a a binding of 2 joysticks and a button using 5 channels.
+
+  &lpuart6 {
+    status = "okay";
+    current-speed = <100000>;
+
+    sbus {
+        compatible = "futaba,sbus";
+        right_stick_x {
+            channel = <1>;
+            label = "Right stick x";
+            type = <INPUT_EV_ABS>;
+            zephyr,code = <INPUT_ABS_RX>;
+        };
+        right_stick_y {
+            channel = <2>;
+            label = "Right stick y";
+            type = <INPUT_EV_ABS>;
+            zephyr,code = <INPUT_ABS_RY>;
+        };
+        left_stick_x {
+            channel = <3>;
+            label = "Left stick x";
+            type = <INPUT_EV_ABS>;
+            zephyr,code = <INPUT_ABS_X>;
+        };
+        left_stick_y {
+            channel = <4>;
+            label = "Left stick y";
+            type = <INPUT_EV_ABS>;
+            zephyr,code = <INPUT_ABS_Y>;
+        };
+        kill_switch {
+            channel = <5>;
+            label = "Kill_switch";
+            type = <INPUT_EV_KEY>;
+            zephyr,code = <INPUT_KEY_0>;
+        };
+    };
+  };
+
+compatible: "futaba,sbus"
+
+include: [base.yaml, uart-device.yaml]
+
+child-binding:
+  description: |
+    SBUS Channel to input-event-code binding
+    INPUT_EV_ABS gives raw input value
+    INPUT_EV_KEY emulates a key
+  properties:
+    channel:
+      type: int
+      required: true
+      description: SBUS input channel
+    label:
+      type: string
+      description: Descriptive name of the key
```

Comment:
I added it because zephyr,code isn't descriptive enough since RC uses different definitions (SA, SC, T1, S1 etc) for that hence a label for keeping track of it.

Reply:
got it, I suggest dropping this and adding comments in the dt file then, that's how it's done in other places